mgcpy/utils/dist_transform.py

```diff
@@ -0,0 +1,66 @@
+import numpy as np
```

Comment:
Make sure they are tested. Coveralls reports a lower coverage percentage. (So that step fails in TravisCI). Plus, look into the PR #33, it already has tests for Mantel and DCORR.